script.js

```diff
@@ -0,0 +1,82 @@
+let Library = null;
+function displayBooks() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.